src/js/components/CheckBox/doc.js

```diff
@@ -54,6 +54,12 @@ export const doc = CheckBox => {
 NOTE: This can only be used with non-toggle components`,
       )
       .defaultValue(false),
+    containerWidth: PropTypes.string
+      .description(
+        `Define a value for the container width and 
+        set value for justify content to space between.`,
+      )
+      .defaultValue(false),
```

Comment:
How is the default value set to boolean when the type of the prop is requesting a string?